lwreg/lwreg.py

```diff
@@ -83,12 +84,18 @@ def register(**kwargs):
         logging.error("Compound already registered")
         sys.exit(1)
 
+## need to think about how to best pass it back
+@cli.command()
```

Comment:
maybe it would make sense to provide an option to allow the user to specify the name of the config file to write?